src/components/Calculator.jsx

```diff
@@ -1,48 +1,131 @@
 import React from 'react';
 import Button from './shared/Button/Button';
+import calculate from './logic/calculate';
 import './style.css';
 
 class Calculator extends React.Component {
   constructor(props) {
     super(props);
     this.state = {};
+    this.onClick = this.onClick.bind(this);
   }
 
+  onClick = (e) => {
+    this.setState((data) => calculate(data, e.target.value));
+  };
+
   render() {
+    const { next, total } = this.state;
     return (
       <div className="wrapper flex align-center justify-center">
         <div className="calculator">
           <div className="answer-display">
-            <p>0</p>
+            <p>{next || total || 0}</p>
           </div>
           <div className="row flex space-btn">
-            <Button className="button gray-bg flex-1" text="AC" />
-            <Button className="button gray-bg flex-1" text="+/-" />
-            <Button className="button gray-bg flex-1" text="%" />
-            <Button className="button orange-bg flex-1" text="/" />
+            <Button
+              className="button gray-bg flex-1"
+              value="AC"
+              onClick={this.onClick}
+            />
+            <Button
+              className="button gray-bg flex-1"
+              value="+/-"
+              onClick={this.onClick}
+            />
+            <Button
+              className="button gray-bg flex-1"
+              value="%"
+              onClick={this.onClick}
+            />
+            <Button
+              className="button orange-bg flex-1"
+              value="/"
+              onClick={this.onClick}
+            />
```

Comment:
- [ ] Dividing a number returns an empty screen. Kindly ensure that your divide ("/") button works.

You can change the value` "/" to "÷"`. Perhaps, It should solve your problem